doc/classes/Object.xml

```diff
@@ -264,6 +264,7 @@
 				[/csharp]
 				[/codeblocks]
 				[b]Note:[/b] The base [Object] defines a few notifications ([constant NOTIFICATION_POSTINITIALIZE] and [constant NOTIFICATION_PREDELETE]). Inheriting classes such as [Node] define a lot more notifications, which are also received by this method.
+				[b]Note:[/b] Unlike other virtual methods, [method _notification] is called automatically for every extended script. This means you should not call the base implementation of [method _notification] via [code]super[/code] in GDScript or its equivalents in other languages.
```

Comment:
Based [on the comment on the other PR](https://github.com/godotengine/godot/pull/107604#discussion_r2150658099) 
```suggestion
				[b]Note:[/b] Unlike other virtual methods, this method is called automatically for every script that overrides it. This means that the base implementation should not be called via [code]super[/code] in GDScript or its equivalents in other languages.
```
Or similar